build/deploy/db_schemas/scd.libsonnet

```diff
@@ -1,12 +1,12 @@
 {
   data:{
-    "upto-v1.0.0-create_initial_version.sql": importstr "rid/upto-v1.0.0-create_initial_version.sql",
-    "upto-v2.0.0-support_api_1_0_0.sql": importstr "rid/upto-v2.0.0-support_api_1_0_0.sql",
-    "upto-v3.0.0-add_inverted_indices.sql": importstr "rid/upto-v3.0.0-add_inverted_indices.sql",
-    "upto-v3.1.0-create_uss_availability.sql": importstr "rid/upto-v3.1.0-create_uss_availability.sql",
-    "downfrom-v3.1.0-remove_uss_availability.sql": importstr "rid/downfrom-v3.1.0-remove_uss_availability.sql",
-    "downfrom-v3.0.0-remove_inverted_indices.sql": importstr "rid/downfrom-v3.0.0-remove_inverted_indices.sql",
-    "downfrom-v2.0.0-remove_api_1_0_0_support.sql": importstr "rid/downfrom-v2.0.0-remove_api_1_0_0_support.sql",
-    "downfrom-v1.0.0-remove_initial_version.sql": importstr "rid/downfrom-v1.0.0-remove_initial_version.sql",
+    "upto-v1.0.0-create_initial_version.sql": importstr "scd/upto-v1.0.0-create_initial_version.sql",
+    "upto-v2.0.0-support_api_1_0_0.sql": importstr "scd/upto-v2.0.0-support_api_1_0_0.sql",
+    "upto-v3.0.0-add_inverted_indices.sql": importstr "scd/upto-v3.0.0-add_inverted_indices.sql",
+    "upto-v3.1.0-create_uss_availability.sql": importstr "scd/upto-v3.1.0-create_uss_availability.sql",
+    "downfrom-v3.1.0-remove_uss_availability.sql": importstr "scd/downfrom-v3.1.0-remove_uss_availability.sql",
+    "downfrom-v3.0.0-remove_inverted_indices.sql": importstr "scd/downfrom-v3.0.0-remove_inverted_indices.sql",
+    "downfrom-v2.0.0-remove_api_1_0_0_support.sql": importstr "scd/downfrom-v2.0.0-remove_api_1_0_0_support.sql",
+    "downfrom-v1.0.0-remove_initial_version.sql": importstr "scd/downfrom-v1.0.0-remove_initial_version.sql",
```

Comment:
Nice catch, we noticed it last week as well.
Since the migration is highly coupled with the code base, wouldn't it make sense to add it to the container image instead of mounting it at runtime ?
This can be addressed later.

Reply:
Agreed; this is static and coupled and should probably be part of the built image.  Added #908.